### DISCUSSION TITLE
Support for Oventrop R-Tronic RT B

### DISCUSSION THREAD

**Discussion by kimzeuner** [mak-gitdev/HA_enoceanmqtt#152] [Request]
I need support on how to integrate my Oventrop R-Tronic RT B in this integration.
In the manual it says, that several EEP`s are supported [manual](https://www.eltako.com/fileadmin/downloads/de/_bedienung/115078080_R-Tronic_ENOCEAN_dt.pdf). For me only the temperature and the setpoint are relevant so i think A5-10-06 should be the correct one. In general the EEP is working well and the values seem are correct but i am not able to send the setpoint from HA to the device. when i publish a new setpoint via mqtt it is shown correct in the HA entity but it is overwritten with the next sending interval from the device. 
On the enocean homepage i have also found the device and here it says that it uses the A5-20-01 EEP but this doesnt work too and also the temperature values are not correct. [link](https://www.enocean-alliance.org/de/product/r-tronic-rt-b/)
Any idea how i can achieve this ? 
Thanks in advance!

**Comment by kloeppb** [comment]
For me these Oventrop Devices are working fine, I have defined them one by one in the the devices file. Get the address from the log file!

Sorry for me only the temp is relevant 

example:

[thermostat_wohnzimmer]
address         = 0x059D39E3
rorg            = 0xA5
func            = 0x10
type            = 0x06

**Comment by kimzeuner** [comment]
Thanks for your reply.
Are you also using the RCON T2P for floor heating or just the thermostat ? And are you using the "normal" RT-B or the enocean variant ? 
As you said, when you are only interested in getting the temperature it works fine, also for me, but i was trying to be able so set the temperature either from within home assistant and sync it with the thermostat or the other way arround. thats why i have bought the enocean variant of the RT-B (inital setup had the normal RT-B which was linked directly with the floor heating controller RCON T2P) as i think it should be possible with them. I think that in the addon there are some teach-in routines missing for my case but i have no idea how to make it work. 
Anyway, again thanks for your reply.

**Comment by kloeppb** [comment]
Well my installation is Enocean R-Tronic RT B and R-con with Wibutler Enocean controller which was setup as part of our new heat pump. As Wibutler is a closed smart home system I am still interested in getting some data out of it into Home Assistant. But the controller is Wibutler.
https://www.oventrop.com/de-DE/produktesysteme/artikeldetails/1150793

Reading data with enoceanMQTT is rather simple as far as I have understood but sending data or commands needs additional definitions you may find examples for switches/buttons in the wiki here. I have not tried this.
